pkg/login/social/okta_oauth_test.go

```diff
@@ -0,0 +1,91 @@
+package social
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"golang.org/x/oauth2"
+
+	"github.com/grafana/grafana/pkg/services/featuremgmt"
+)
+
+const testOktaUserRawJSON = `{
+  "sub": "00u7dfyci33dabAc25d7",
```

Comment:
not sure if I should include our setup for the tests due to security. But it is only a id and some email configurations that is needed.
@Jguer

Reply:
this is from https://github.com/grafana/grafana-authnz-team/wiki/Development-Environment#okta-oauth

Just our configuration. the way I could make it work.
il have a look at the jwt. see the other conversation https://github.com/grafana/grafana/pull/62547#discussion_r1091665942

Reply:
Right now that you committed and pushed it it's public anyway. Except `client_secret` these are not sensitive. You can still replace them with placeholder values